robottelo/ui/locators/base.py

```diff
@@ -1279,6 +1279,9 @@
     "rh.prd_expander": (
         By.XPATH, ("//div[@id='ui-tabs-1']//td[contains(.,'%s')]"
                    "/span[@class='expander']")),
+    "rh.ostree_prd_expander": (
+        By.XPATH, ("//div[@id='ui-tabs-7']//td[contains(.,'%s')]"
```

Comment:
@oshtaier : Thank you for raising this. I tried all possible combinations but that doesn't work without `ui-tabs-x` in locators. Here is little bit more explanation around this.

So if you see the difference between `rh.prd_expander` and `rh.ostree_prd_expander` locators is that both are using `ui-tabs-x` based on the tab number under 'Red Hat Repositories` page.  The first tab is of `RPM` based repos (so locators is using ui-tabs-1)and 7th tab is of `ostree` repos (so locators is using ui-tabs-7).

I really wanted to have a unified locator for all tabs, but I couldn't succeed. If anyone has any other idea to get this done to make it unified, please suggest here. I'll be more than happy to update the PR.  Thank you.